.goreleaser.yml

```diff
@@ -0,0 +1,18 @@
+project_name: 'mm-geofeed-verifier'
```

Comment:
This looks like the same as the default name that will be chosen by GoReleaser - is there a reason you chose to specify it manually?

Reply:
I think was leaning towards 'explicit is always better' that day. Is this a case where letting defaults go is more the norm?

Reply:
My view is when it comes to thinks like a `.goreleaser.yml` I think of most of it as boilerplate and I like to keep it as minimal as possible. 

No code is the best code, my pappy always used to say.

I don't think I'll not merge if you have differing views!